content/working_with/manager/using-secrets.md

```diff
@@ -63,6 +63,17 @@ For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.m
 
 ![Create Secret]( /images/manager/create_secret_dialog.png )
 
+### Creating a secret with Secret Provider from the {{< param cfy_console_name >}}
+1. Go to the [Secrets page]({{< relref "working_with/console/pages/secrets-page.md" >}}).
+2. Click **Create** in the [Secret Store Management widget]({{< relref "working_with/console/widgets/secrets.md" >}}).
+3. Insert the secret key
+4. Click on the checkbox "Retrieve the secret value from a secret provider"
```

Comment:
What do you think about adding something like (to be updated before committing as I didn't include correct link)?:

```suggestion
4. Click on the checkbox "Retrieve the secret value from a secret provider" (you must have at least one secret provider defined in {{< param product_name >}} to be able to use this option, see [Secret Providers widget](link to Secrets Provider widget) for details)
```

Reply:
As mentioned in the original comment - link to be updated 🙂

Reply:
didn't notice that 🙈 https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2524/commits/ef89980c9af79fd8763678a8200af9e46696a842 updated link and one small typo :)